includes/User.php

```diff
@@ -5007,6 +5007,8 @@ protected function saveOptions() {
 
 		$dbw->upsert('user_properties', $insertRows, [], self::$PROPERTY_UPSERT_SET_BLOCK);
 
+		$this->userPreferences()->save($this->getId());
```

Comment:
Hmm I wonder if this should be done in `User->saveSettings()` right after the call to `$this->saveOptions()`? since it's really outside of options now